### DISCUSSION TITLE
Any MikTeX for Windows 7?

### DISCUSSION THREAD

**Discussion by vgiv** [MiKTeX/miktex#1258] [Q&A]
Hi!
Is there any MikTeX distribution that works under Windows 7? A have tried a lot of versions and they:
1) either do not support W7
2) or report errors when I try to install packages from a remote repository (and I do not understand how to install a local one, all my attempts lead to "Damaged configuration file" error etc).

Any advice?

**Comment by VassilisM** [comment]
Hi. Have you found a solution?
I still have an old installer "basic-miktex-21.2-x64" working on Windows 7. Running update, it broke it :-P
So, running an old version not able to update. Don't know about installing any packages though. UPDATE: It found all that it needed for my old TeX file.

**Comment by vgiv** [comment]
> Hi. Have you found a solution? I still have an old installer "basic-miktex-21.2-x64" working on Windows 7. Running update, it broke it :-P So, running an old version not able to update. Don't know about installing any packages though. UPDATE: It found all that it needed for my old TeX file.

No. I haven't found any solution and migrated to texlive eventually.

**Comment by josephernest** [comment]
I have tried with `basic-miktex-2.9.7386-x64.exe` from a few years ago. 

It works.

But it fails when we want to install a new package. Even when using a *local filesystem repository mirror* from https://www.ctan.org/tex-archive/systems/win32/miktex/tm/packages.

@edocevoli would you have an idea of solution for Windows 7? There are many cases for which upgrading to Win10 is not possible (computer too old, or computer needs to keep Win7 because of other software that only work on Win7, etc.).
